src/main/java/com/starrocks/connector/flink/manager/StarRocksStreamLoadVisitor.java

```diff
@@ -183,7 +183,10 @@ private String getAvailableHost() {
         List<String> hostList = sinkOptions.getLoadUrlList();
         long tmp = pos + hostList.size();
         while (pos < tmp) {
-            String host = "http://" + hostList.get((int) (pos % hostList.size()));
+            String host = hostList.get((int) (pos % hostList.size()));
+            if (host != null && !host.startsWith("http")) {
```

Comment:
How about move this logic to `StarRocksSinkOptions#getLoadUrlList` to reduce code duplication? `getLoadUrlList` will return urls already prefixed with http or https.